angelleye-includes/angelleye-session-functions.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
Fantastic Job @kcppdevelopers 
I like this function file to handle the sessions.